base.go

```diff
@@ -36,6 +37,75 @@ func GetBaseLayer(o BaseLayerOpts) error {
 	}
 }
 
+func getOci(o BaseLayerOpts) error {
+	url, err := o.Layer.From.ParseTag()
+	if err != nil {
+		return err
+	}
+	// url is "ocidir:image"
+	res := strings.SplitN(url, ":", 2)
+	fullImage := res[1]
+	// image may be "imagebase_tag" i.e. boot_0.1.3
+	res = strings.SplitN(fullImage, "_", 2)
+	imageBase := fullImage
+	tag := ""
+	if len(res) == 2 {
+		imageBase = res[0]
+		tag = res[1]
+	}
+
+	cacheDir := path.Join(o.Config.StackerDir, "layer-bases", tag)
+	if err := os.MkdirAll(cacheDir, 0755); err != nil {
+		return err
+	}
+
+	cmd := exec.Command(
+		"skopeo",
```

Comment:
I think you need an `--insecure-policy` here to keep everyone from having to add /etc/container/policy.json to their systems.